remotepin.go

```diff
@@ -7,14 +7,21 @@ const (
 )
 
 type Pinning struct {
-	RemoteServices map[string]RemotePinningService
+	MFSRepinInterval string // in ns, us, ms, s, m, h
+	RemoteServices   map[string]RemotePinningService
 }
 
 type RemotePinningService struct {
-	Api RemotePinningServiceApi
+	Api      RemotePinningServiceApi
+	Policies RemotePinningServicePolicies
 }
 
 type RemotePinningServiceApi struct {
 	Endpoint string
 	Key      string
 }
+
+type RemotePinningServicePolicies struct {
+	// PinMFS enables watching for changes in MFS and re-pinning the MFS root cid whenever a change occurs.
+	PinMFS *bool
```

Comment:
WDYT about making each policy a struct. This could allow us to add more options per policy without polluting the namespace.

Reply:
I think we should go one step beyond that and  have reusable  `Policies` struct – see https://github.com/ipfs/go-ipfs-config/pull/116/files#r539339668